kolibri/core/tasks/scheduler.py

```diff
@@ -182,6 +182,13 @@ def get_jobs(self):
             scheduled_jobs = self._ns_query(s).all()
             return [Job.from_json(o.saved_job) for o in scheduled_jobs]
 
+    def test_table_readable(self):
+        """
+        Do a quick query to raise errors if the database is unusable.
+        """
+        with self.session_scope() as s:
```

Comment:
I am not sure I am able to comprehend this  particular part well. Other than that, everything looks good!

Reply:
This is our helper context manager for creating a SQLAlchemy Session. https://docs.sqlalchemy.org/en/14/orm/session_basics.html#basics-of-using-a-session
The context manager handles the commit/rollback pattern described here: https://docs.sqlalchemy.org/en/14/orm/session_basics.html#framing-out-a-begin-commit-rollback-block

The source code for the `session_scope` context manager is here: https://github.com/learningequality/kolibri/blob/release-v0.15.x/kolibri/core/tasks/storage.py#L62